aten/src/THC/THCAllocator.h

```diff
@@ -4,7 +4,6 @@
 #include "THCGeneral.h"
 
 THC_API THAllocator* getTHCudaHostAllocator(void);
-THC_API THAllocator* getTHCUVAAllocator(void);
```

Comment:
@nicolasvasilache Is tensor comprehensions using this still? I know you mentioned doing some experiments with this a while ago.

Reply:
Maybe let's play it safe and keep the UVA Allocator for now?

Reply:
No need, we can kill it.

Reply:
Cool.